.goreleaser.template.yml

```diff
@@ -0,0 +1,51 @@
+env:
+- GOPROXY=https://proxy.golang.org|direct
+- GO111MODULE=on
+- CGO_ENABLED=0
+- PKG=github.com/operator-framework/operator-controller/internal/version
+before:
+  hooks:
+    - go mod tidy
+    - go mod download
+builds:
+  - id: operator-controller
+    main: ./
+    binary: bin/manager
+    tags: $GO_BUILD_TAGS
+    goos:
+    - linux
+    ldflags:
+    - -X {{ .Env.PKG }}.GitCommit={{ .ShortCommit }}
+dockers:
+- image_templates:
+  - "{{ .Env.IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}"
+  dockerfile: Dockerfile
+  goos: linux
+docker_manifests:
+- name_template: "{{ .Env.IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}"
+  image_templates:
+  - "{{ .Env.IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}"
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
+changelog:
+  use: github-native
+  skip: $DISABLE_RELEASE_PIPELINE
+release:
+  disable: $DISABLE_RELEASE_PIPELINE
```

Comment:
I'm trying to remember why we do this convoluted envsubst thing rather than use the `--snapshot` flag when invoking goreleaser.

Reply:
Ah I remember: we want to build/push images/manifests -- but NOT release -- for builds of the main branch.

It seems like this problem is half solved. It's possible to template `release.disable`, but still not possible to template `changelog.skip`.

I opened an issue in goreleaser to see if we can make `changelog.skip` templatable as well: https://github.com/goreleaser/goreleaser/issues/3828

Reply:
Sweet. The maintainer already merged a PR that fixes this. Once there's a release, we can directly reference `{{ .Env.DISABLE_RELEASE_PIPELINE }}` as a template value in the actual goreleaser config file. No need for the envsubst shenanigans after that.